sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 1.0.4 (Unreleased)
 
+- A new type is exported, `ChallengeCallbackOptions`, which contains `scopes`, a string `claims` that might contain a challenge, the previous token used on a request `previousToken`, a `getToken` function, which calls to the underlying credential's `getToken` with a smart approach that minimizes unnecessary network requests, the pipeline request `request`, and a `setAuthorizationHeader` which makes it easier to set the token on the pipeline request.
```

Comment:
These notes tell a slightly confusing story, can we start with a bullet point about how CAE is now supported and a link to what it is, followed by sub-bullets about the details?

Reply:
Also might be easier to create a sample and link to it rather than describing the full capabilities in prose